internal/workflow/internal/reconcile_test.go

```diff
@@ -0,0 +1,135 @@
+package internal_test
```

Comment:
im not fond of `internal_test` files that use dot notation import for the package it's testing. I don't see the value. Not anything to hold up the PR, just wanted to voice it. : )

Reply:
Do you mean specifically for `internal` packages, or do you mean generally the `_test` idiom with dot import?